courses/free-courses-hi.md

```diff
@@ -428,6 +429,7 @@
 #### jQuery
 
 * [jQuery Tutorial for beginners in Hindi 2020](https://youtube.com/playlist?list=PL-6FWL4WVVWXmWe_HnPG0rBQmmJfGsTKS) - CSEtutorials
+* [jQuery Tutorial for beginners](https://youtu.be/YFlx1C8XwR0?si=EUxv49jO_iZ3EcWX) - CodeWithHarry
```

Comment:
We prefer 'playlist' links for YouTube contributions and provide complete links; shortened links are not accepted.